src/Components/Nav.scss

```
@@ -0,0 +1,80 @@
+@import "../Styles/common.scss";
+
+.Nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  width: 100%;
+  height: 120px;
+  padding: 0 4.75% 0;
+  background-color: $brick;
+
+  .siteLogo {
+    a {
+      background-image: url("//hince.co.kr/assets/images/hince-logo-sprite.png");
+      background-size: 98px 70px;
+      background-position-y: -35px;
+      display: block;
+      width: 98px;
+      height: 35px;
+      margin-top: 0px;
+    }
+  }
+
+  .siteMenu {
+    ul {
+      display: flex;
+      position: relative;
+
+      li {
+        @include list;
+        padding: 7px 30px;
+
+        a {
+          text-decoration: none;
+          font-size: 17px;
+          font-weight: 600;
+          padding: 5px 7px;
+          color: $white;
+        }
+      }
+    }
+  }
+
+  .siteExtra {
+    float: right;
+
+    ul {
+      display: flex;
+
+      li {
+        @include list;
+        background-image: url("//hince.co.kr/assets/images/hince-icon-sprite-185484.png");
```

```suggestion
        background-image: url("http://hince.co.kr/assets/images/hince-icon-sprite-185484.png");
```
http://로 시작해야 하는 내용 아닐까요?